pkg/types/interfacetests/chain_components_interface_tests.go

```diff
@@ -16,6 +16,40 @@ import (
 	"github.com/smartcontractkit/chainlink-common/pkg/values"
 )
 
+// GetLatestValue method
+const GetLatestValueAsValuesDotValue = "Gets the latest value as a values.Value"
```

Comment:
Since the package is not limited to contract reader, should these be qualified?
```suggestion
const ContractReaderGetLatestValueAsValuesDotValue = "Gets the latest value as a values.Value"
```
I think that might actually help readability somewhat, since it is recognizable as the test idiom.

Reply:
Done